Polyfills/AbortController/Source/AbortSignal.cpp

```diff
@@ -93,6 +105,19 @@ namespace Babylon::Polyfills::Internal
             }
         }
     }
+
+    void AbortSignal::RaiseEvent(const char* eventType)
+    {
+        auto it = m_eventHandlerRefs.find(eventType);
+        if (it != m_eventHandlerRefs.end())
+        {
+            const auto& eventHandlerRefs = it->second;
+            for (const auto& eventHandlerRef : eventHandlerRefs)
+            {
+                eventHandlerRef.Call({});
```

Comment:
Do we need to check that `eventHandlerRef` is not null / undefined?

Reply:
This was a function already existing from XMLHttpRequest. I can't imagine that someone is going to .addEventListener('abort', null) but I suppose it might be a possibility?